cluster/gke/util.sh

```diff
@@ -193,6 +193,22 @@ function kube-up() {
   # Bring up the cluster.
   "${GCLOUD}" ${CMD_GROUP:-} container clusters create "${CLUSTER_NAME}" "${create_args[@]}"
 
+  if [[ "${FEDERATION:-}" == "true" ]]; then
+    # Create a kubeconfig with credentials for this apiserver. We will later use
+    # this kubeconfig to create a secret which the federation control plane can
+    # use to talk to this apiserver.
+    KUBECONFIG=${KUBECONFIG:-${HOME}/.kube/config}
+    KUBECONFIG_DIR=$(dirname $KUBECONFIG)
+    CONTEXT=$($KUBECTL config current-context)
+    DEST_KUBECONFIG="${KUBECONFIG_DIR}/federation/kubernetes-apiserver/${CONTEXT}/kubeconfig"
+    mkdir -p $(dirname $DEST_KUBECONFIG) >&2
+    # TODO: Original kubeconfig can contain credential information about
```

Comment:
gcloud respects the KUBECONFIG env. So if you want to create a kubeconfig file with a single cluster's info, just
```
export KUBECONFIG=/path/to/standalone/kubeconfig
gcloud container clusters get-credentials mycluster

# do stuff with /path/to/standalone/kubeconfig
```